utils/mock-data-loader/bss/eligibilityCriteria.js

```diff
@@ -266,4 +266,98 @@ module.exports = [
+      {
+        id: 12,
+        description:
+          'The cover period for each Transaction does not exceed 5 years, or such other period approved by UKEF (that has not lapsed or been withdrawn) in relation to bonds and/or loans for this Obligor.',
+      },
+      {
+        id: 13,
+        description:
+          'The total UKEF exposure, across all short-term schemes (including bond support, export working capital and general export facility transactions), for this Obligor (including this Transaction) does not exceed £5 million, or such other limit approved by UKEF (that has not lapsed or been withdrawn).',
+      },
+      {
+        id: 14,
+        description:
+          'For a bond Transaction, the bond has not yet been issued or, where the bond has been issued, this was done no more than 3 months prior to the submission of this Inclusion Notice. For a loan Transaction, the loan has not yet been advanced.',
+      },
+      {
+        id: 15,
+        description: 'The Requested Cover Start Date is no more than three months from the date of submission.',
+      },
+      {
+        id: 16,
+        description:
+          'The Supplier has confirmed in its Supplier Declaration that the Supply Contract does not involve any of the following Controlled Sectors: sharp arms defence, nuclear, radiological, biological, human cloning, pornography, tobacco, gambling, coal, oil, gas or fossil fuel energy and the Bank is not aware that any of the information contained within it is inaccurate.',
+      },
+      {
+        id: 17,
+        description:
+          'The Bank has completed its Bank Due Diligence to its satisfaction in accordance with its policies and procedures without having to escalate to any Relevant Person.',
+      },
+      {
+        id: 18,
+        description:
+          "The fees, interest rate and/or risk margin apply to the whole Cover Period, and have been set in accordance with the Bank's normal pricing policies and include, if any, pricing requirements set by UKEF.",
+      },
+    ],
+  },
+  {
+    version: 7,
+    product: 'BSS/EWCS',
+    isInDraft: false,
+    createdAt: 1702061978881,
+    criteria: [
+      {
+        id: 11,
+        description:
+          'The Supplier has confirmed in its Supplier Declaration that the Supply Contract does not involve agents and the Bank is not aware that any of the information contained within it is inaccurate.',
+      },
+      {
+        id: 12,
+        description:
+          'The period between the Cover Start Date and the Cover End Date does not exceed for a Bond, the Bond Maximum Cover Period; and for a Loan, the Loan Maximum Cover Period',
+      },
+      {
+        id: 13,
+        description:
+          'The Covered Bank Exposure under the Transaction converted (as at the date this representation is made) for this purpose into the Base Currency) is not more than the lesser of the Available Facility; and the Available Obligor Covered Exposure Limit',
```

Comment:
```suggestion
          'The Covered Bank Exposure under the Transaction (converted (as at the date this representation is made) for this purpose into the Base Currency) is not more than the lesser of: the Available Facility; and the Available Obligor Covered Exposure Limit.',
```
I think we should:
- add the missing opening bracket before `converted`
- have a full stop at the end
- keep the colon before the parts of the text that are bullet points in the TFM version of the text

What do you think?
If we change it then we need to make the same change in all the mock data that has this EC as well